eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -25,12 +28,17 @@ class Report:
     One instance of this class is instantiated in the running of the pipeline.
     """
 
-    def __init__(self, unavailable_efo=None):
+    def __init__(self, trait_mappings=None, unavailable_efo=None):
```

Comment:
Making `trait_mappings={}` and `unavailable_efo=set()` the default won't imply an extra cost when calling `deepcopy` and it would remove the need for if-else branches.